File: Java/074. Fatorial/Fatorial.java
import java.util.Scanner;
public class Fatorial
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        long n = scan.nextInt();
        long fatorial = 1;
        for (long i = 1; i <= n; i++)
        {
            fatorial *= i;
        }
        System.out.println(fatorial);
        scan.close();
    }
}

File: C++/074. Fatorial/Fatorial.cpp
#include <iostream>
int main() 
{
    int N;
    std::cin >> N;
    long long fatorial = 1;
    if (N == 0)
    {
        std::cout << fatorial << std::endl;
        return 0;
    }
    for (int i = 1; i <= N; ++i) {
        fatorial *= i;
    }
    std::cout << fatorial << std::endl;
    return 0;
}